### DISCUSSION TITLE
ScrollArea questions

### DISCUSSION THREAD

**Discussion by jkvargas** [emilk/egui#2561] [Q&A]
Hi,
Where should I look in the code what defines the position of the scroll area.
And the interaction with the mouse input?

I am interested on syncing two scroll areas in order to write some sort of gantt component.
ex. https://github.com/neuronetio/gantt-schedule-timeline-calendar

**Comment by NickNick** [comment]
It's not what you are asking, but for a similar time-liney view I found a lot of inspiring code in https://github.com/EmbarkStudios/puffin. If I remember correctly, it is a lot of custom drawing (which sounds like a bad thing in my book) but I found it easy enough to use.

Otherwise, check the return value of ScrollArea::show, which has .state.offset, and then 'ui' has scroll_to_offset? Maybe that could work?
